src/lib.rs

```diff
@@ -129,6 +129,7 @@ pub fn run_cmd(cmd: Command) -> Result<(), Error> {
         }
 
         SubCommand::Clean {} => commands::clean_venv(&context?),
+        SubCommand::Clean {} => commands::clean_venv(context?),
```

Comment:
Nit: It'd look cleaner to remove this line out of the commit and rebase! Currently it's fixed in the next commit